### DISCUSSION TITLE
Any Suggestion(s)?

### DISCUSSION THREAD

**Discussion by dhruvkrishnavaid** [community/community#21482] [Other Feature Feedback, Questions, & Ideas]
<p>I have made my <a href="https://dhruvkrishnavaid.github.io" rel="nofollow noopener">portfolio website</a>. It has some free cybersecurity tools that everyone must adopt. Any suggestions on improving this?</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="dhruvkrishnavaid" data-post="1" data-topic="127943">
<div class="title">
<div class="quote-controls"></div>
 dhruvkrishnavaid:</div>
<blockquote>
<p>Any suggestions on improving this?</p>
</blockquote>
</aside>
<p>Great website. If I may point out an issue, in the <a href="https://dhruvkrishnavaid.github.io/cyber-security.html" rel="nofollow noopener">Cyber-Security Tools</a> page, the section <em>The scale of the cyber threat</em> with the Matrix-styled background is a bit hard to read — increasing font size and boldness might improve, but you might also want to improve color contrast there (or tweak the BG image).</p>
<p>You might use these online tools to ensure that the text foreground (and size) has a good contrast with the background, and obtain the actual accessibility scores of the current website:</p>
<ul>
<li><a href="https://webaim.org/resources/contrastchecker/" rel="nofollow noopener">https://webaim.org/resources/contrastchecker/</a></li>
<li><a href="https://contrastchecker.com/" rel="nofollow noopener">https://contrastchecker.com/</a></li>
</ul>
<p>Some articles on the topic:</p>
<ul>
<li><a href="https://webaim.org/articles/contrast/" rel="nofollow noopener">WebAIM » Understanding WCAG 2 Contrast and Color Requirements</a></li>
<li><a href="https://webaim.org/resources/evalquickref/" rel="nofollow noopener">WebAIM » Testing Web Content for Accessibility</a></li>
</ul>
<p>Also, always try to test text/BG color combinations for color-blindness accessibility (green being a color that is often perceived differently by users affected by color blindness).</p>
<p>I can’t think of a link to an online tool that does that, I usually develop color schemes using <a href="https://www.tigercolor.com/ColorImpact.htm" rel="nofollow noopener">ColorImpact 4</a>, which features a color blindness simulator that allow you to see any color scheme (and test images and texts) as they are perceived by the various types of color blindness.</p>

**Comment by SethClydesdale** [comment]
<aside class="quote no-group" data-username="tajmone" data-post="2" data-topic="127943">
<div class="title">
<div class="quote-controls"></div>
 tajmone:</div>
<blockquote>
<p>Also, always try to test text/BG color combinations for color-blindness accessibility (green being a color that is often perceived differently by users affected by color blindness).</p>
</blockquote>
</aside>
<p>I just want to add that if you’re using a Chromium-based browser (Chrome, Edge, etc.) you can emulate some vision deficiencies under Rendering in the devtools. 3 dots &gt; More tools &gt; Rendering, then scroll to the very bottom for the emulation options.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0be1533285d6c17b9dee5137354dda116e8d39ed.jpeg" data-download-href="/uploads/short-url/1H60xAIcKCWUwx10Bt4osZSVQK1.jpeg?dl=1" title="vision-emulation.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10122549/181064966-ff12cffd-a4e2-479b-8889-72feedd3f4be.jpeg" alt="vision-emulation.PNG" data-base62-sha1="1H60xAIcKCWUwx10Bt4osZSVQK1" width="690" height="353" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/0be1533285d6c17b9dee5137354dda116e8d39ed_2_690x353.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/0/0be1533285d6c17b9dee5137354dda116e8d39ed_2_1035x529.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/0be1533285d6c17b9dee5137354dda116e8d39ed.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0be1533285d6c17b9dee5137354dda116e8d39ed_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">vision-emulation.PNG</span><span class="informations">1155×592 188 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>More info:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/10122549/181064968-868dc491-1aac-40ad-ba32-9574eca5a147.png" class="site-icon" width="32" height="32">
      <a href="https://developers.google.com/web/updates/2020/03/devtools" target="_blank" rel="nofollow noopener">Google Developers</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/388;"><img src="https://user-images.githubusercontent.com/10122549/181064969-d6d01a54-add9-42ba-8316-ca0194096ae6.png" class="thumbnail" width="690" height="388" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/037e38456d9b51512ec50d968a36277eb89f28f2_2_10x10.png"></div>

<h3><a href="https://developers.google.com/web/updates/2020/03/devtools" target="_blank" rel="nofollow noopener">What's New In DevTools (Chrome 83)  |  Web  |  Google...</a></h3>

<p>Emulate color vision deficiencies, emulate locales, COOP and COEP debugging, and lots more.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by dhruvkrishnavaid** [comment]
<p><a class="mention" href="/SethClydesdale">@SethClydesdale</a> <a class="mention" href="/tajmone">@tajmone</a><br>
Thank you both for your precious help…<br>
I have tweaked the BG Image (added multiple vignettes and reduced brightness).</p>
<p>Sorry for disturbing, but I wanted to ask if the section is readable now.</p>
<p>I also checked for vision deficiency emulation. I had no issue in reading the text with all the emulations (except the blurred visoin).</p>
<p>Thanks for your help.</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="dhruvkrishnavaid" data-post="4" data-topic="127943">
<div class="title">
<div class="quote-controls"></div>
 dhruvkrishnavaid:</div>
<blockquote>
<p>I wanted to ask if the section is readable now.</p>
</blockquote>
</aside>
<p>The situation has improved but there are still some spots where the BG image bright parts tend to “absorb” the text, e.g. around the “With the scale of the cyber threat …” text.</p>
<p>The section it’s not unreadable, but it definitely requires some effort on the reader’s side to focus on the text, which might be a deterrent for visitors who have vision difficulties. In my case, my sight is not good as it used be, and even with monitor dedicated glasses I find it a bit strainful to read that section. Since I spend many hours in front of the PC (which is by itself a strain on the eyes) I tend to skip website/pages that have hard-to-read color schemes.</p>
<p>I think that if you add a think black contour to the text in that section it would become much easier to read (like a very thin shadow/glow that separates the white text from the bright spots of the BG image).</p>

**Comment by tajmone** [comment, marked as answer]
<aside class="quote no-group" data-username="SethClydesdale" data-post="3" data-topic="127943">
<div class="title">
<div class="quote-controls"></div>
 SethClydesdale:</div>
<blockquote>
<p>if you’re using a Chromium-based browser (Chrome, Edge, etc.) you can emulate some vision deficiencies under Rendering in the devtools.</p>
</blockquote>
</aside>
<p>Thanks for the tip <a class="mention" href="/SethClydesdale">@SethClydesdale</a>, I didn’t realize Google had introduced these accessibility tool. That’s really cool, web/GUIs accessibility is a really important topic, and unfortunately an often neglected one too.</p>
<p>I’ve got into the habit of handling all my color schemes design and test work in ColorImpact because it supports color profiles, whereas historically most browser didn’t support color profiles at all —<br>
nowadays Chrome and a couple of other browsers support color profiles, but some browser still don’t. Many applications don’t support color profiles either, including image previewers, so when it comes to colors fine-tuning I’m always careful to work with apps that I’m sure can handle colors correctly.</p>
<p>In case anyone needs it, I’ve posted an image to quickly test if an app supports color profiles:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/11064292/181064973-2c281924-eed2-44c1-a1e1-96381d1a3cc3.png" class="site-icon" width="16" height="16">
      <a href="https://community.cosmigo.com/t/help-on-export-colors/642/8" target="_blank" rel="nofollow noopener" title="10:47AM - 20 September 2019">cosmigo – 20 Sep 19</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11064292/181064974-b535d8ba-8bfc-429c-929e-9fb9a1abfbbf.png" class="thumbnail onebox-avatar" width="500" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c29822a8289cf420eb01a539415eb05bbb8dffb0_2_10x10.png">

<h3><a href="https://community.cosmigo.com/t/help-on-export-colors/642/8" target="_blank" rel="nofollow noopener">Help on export colors</a></h3>

<p>@iliasboufidis, you should use this image to understand if an application supports color profiles:                                                  Its color profile is tweaked so that blue and red are actually swapped on any app that doesn’t...</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
